components/service/glean/src/main/java/mozilla/components/service/glean/storages/GenericScalarStorageEngine.kt

```diff
@@ -122,15 +130,54 @@ internal abstract class GenericScalarStorageEngine<ScalarType> : StorageEngine {
     }
 
     /**
-     * Ensures that the user lifetime metrics (in [userLifetimeStorage]) is
+     * Gets SharedPreference for a particular lifetime
+     *
+     * @param lifetime [Lifetime] for which to get the preferences for
+     * @return [SharedPreferences]
```

Comment:
nit: this is incorrect: we either return a valid `[SharedPreferences]` or `null`